packages/models-library/src/models_library/callbacks_mapping.py

```diff
@@ -36,6 +39,28 @@ class CallbacksMapping(BaseModel):
             "user services are allowed"
         ),
     )
+    inactivity: UserServiceCommand | None = Field(
+        None,
+        description=(
+            "command used to figure out for how much time the "
+            "user service(s) were inactive for"
+        ),
+    )
+
+    @validator("inactivity")
+    @classmethod
+    def ensure_inactivity_timeout_is_capped(
+        cls, v: UserServiceCommand
+    ) -> UserServiceCommand:
+        if v is not None and (
+            v.timeout < TIMEOUT_MIN or v.timeout > INACTIVITY_TIMEOUT_CAP
+        ):
+            msg = (
+                f"Constraint not respected for inactivity timeout={v.timeout}: "
```

Comment:
question: why do you cap the timeout for this command and not the others?

Reply:
While the other commands depend on what the service creator wants them to do. 
Here, oSPARC, defines how this needs to be implemented.

In this situation we want this command to respond in a certain time window. The value that we want back is a number which should be already stored in memory and tis should not take more than a few milliseconds to respond.

Reply:
As a service owner when you define other commands, you still have to specify a timeout parameter. You are free to specify how long the timeout is, but the timeout parameter is always required.

In this case, for the inactivity, you specify it but are constraint to pick a vale int the interval `[1, 5]` (seconds).

Reply:
but then as a user I can define a timeout of 10 hours if I like to?

Reply:
Yes, that is a possibility at the moment.